src/components/ContinuationIn/ExtraproRegistration.vue

```diff
@@ -0,0 +1,331 @@
+        </v-row>
+      </template>
+
+      <template v-else>
+        <v-row
+          class="mt-2"
+          no-gutters
+        >
+          <v-col
+            cols="12"
+            sm="3"
+          >
+            <!-- empty column to line up with Business Information above -->
+          </v-col>
+
+          <v-col
+            cols="12"
+            sm="9"
+          >
+            <MessageBox color="red">
+              <strong>Important:</strong> This extraprovincial registration is historical.
+              Please contact BC Registries staff.
+            </MessageBox>
+
+            <RegistriesContactInfo class="mt-6" />
+          </v-col>
+        </v-row>
+      </template>
+    </template>
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Emit, Vue, Watch } from 'vue-property-decorator'
+import { Action, Getter } from 'pinia-class'
+import { useStore } from '@/store/store'
+import { isEqual } from 'lodash'
+import { getName } from 'country-list'
+import { BusinessLookupResultIF, EmptyBusinessLookup } from '@/interfaces'
+import ExtraproBusinessLookup from './ExtraproBusinessLookup.vue'
+import MessageBox from '@/components/common/MessageBox.vue'
+import RegistriesContactInfo from '@/components/common/RegistriesContactInfo.vue'
+
+@Component({
+  components: {
+    ExtraproBusinessLookup,
+    MessageBox,
+    RegistriesContactInfo
+  }
+})
+export default class ExtraproRegistration extends Vue {
+  @Getter(useStore) getContinuationInBusinessInfo!: any
+  @Getter(useStore) getShowErrors!: boolean
+
+  @Action(useStore) setContinuationInBusinessInfo!: (x: any) => void
+
+  // Local properties
+  active = false
+  business = null // FUTURE: update BusinessLookupResultIF with new properties
+  checkbox = false
+
+  readonly checkboxLabel = 'I understand and acknowledge that the extraprovincial registration of this' +
+    ' business will be cancelled and made historical in the B.C. Registry once the continuation in' +
+    ' application has been submitted.'
+
+  /** Whether we have a looked-up business. */
+  get haveLookupBusiness (): boolean {
+    return !isEqual(this.business, EmptyBusinessLookup)
+  }
+
+  /** The identifier. */
+  get identifier (): string {
+    return this.business.identifier
+  }
+
+  /** The jurisdiction. */
+  get jurisdiction (): string {
+    const hj = this.business.homeJurisdiction
+    if (hj?.country) {
+      const country = getName(hj.country)
+      const region = (hj.region === 'FEDERAL' ? 'Federal' : hj.region)
+      if (region) return `${region}, ${country}`
+      return country
+    }
+    return null
+  }
+
+  /** The business name. */
+  get businessName (): string {
+    return this.business.name
+  }
+
+  /** The business number. */
+  get businessNumber (): string {
+    return this.business.bn
+  }
+
+  /** The incorporation date. */
+  get incorporationDate (): string {
+    // FUTURE: format date if needed
+    return this.business.registrationDate
+  }
```

Comment:
I have **no idea** what the registration date will look like from business search.

An example I got from David Roberts looks like this (in COLIN):

![image](https://github.com/bcgov/business-create-ui/assets/10002889/3471f738-7e4b-4ab3-9208-2251a1e52e7d)